src/platform/plugins/shared/discover/public/application/main/components/top_nav/app_menu_actions/get_share.tsx

```diff
@@ -71,7 +77,15 @@ export const getShareAppMenuItem = ({
       refreshInterval,
     };
 
-    // TODO: for a persisted saved search, add the current tab ID to the params
+    const tabsEnabled = core.featureFlags.getBooleanValue(TABS_ENABLED_FEATURE_FLAG_KEY, false);
```

Comment:
```suggestion
    const tabsEnabled = discoverFeatureFlags.getTabsEnabled()
```
To make sure we get the right default.